src/relevance_feedback/relevance_feedback.py

```diff
@@ -0,0 +1,81 @@
+import cv2, os
+import numpy as np
+from scipy.stats import moment
+from itertools import repeat
+import pickle
+
+import os,sys,inspect
+sys.path.insert(0, '../backend/')
+import singular_value_decomposition
+from database_connection import DatabaseConnection
+from utils import get_most_m_similar_images
+
+class RelevanceFeedback:
+
+	def __init__(self):
+		self.database_connection = DatabaseConnection()
+		self.conn = self.database_connection.get_db_connection()
+		print('Initiating RelevanceFeedback')
+
+	def compute_new_query_vector(self,q_old,relevant_items,irrel_items,alpha=0.5,beta=0.45,gamma=0.05):
```

Comment:
Any resource I can read on choosing alpha, beta and gamma values?

Reply:
No source as such... we need to check our output by hit and trial